metrics/grafana/pd.json

```diff
@@ -1655,12 +1655,20 @@
           ],
           "targets": [
             {
-              "expr": "pd_cluster_placement_status{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\"}",
+              "expr": "sum(pd_cluster_placement_status{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\"}) by (name)",
```

Comment:
Will it affect the compatibility?

Reply:
old pd with old `metrics.json` is was
<img width="1141" alt="image" src="https://github.com/tikv/pd/assets/53859786/0cc9a19f-3654-40bd-b4c3-34a1724ae92c">
old pd with new `metrics.json` is was
<img width="1096" alt="image" src="https://github.com/tikv/pd/assets/53859786/3d2c896c-87fb-424f-ae5b-5d8db0585b24">
And I **hide** the old expression in the new `metrics.json` to avoid breaking compatibility.
<img width="1108" alt="image" src="https://github.com/tikv/pd/assets/53859786/37905f43-e0d3-49a3-9bf8-3ae897a1a4c9">